Gradata/src/gradata/_systemd_installer.py

```
@@ -0,0 +1,137 @@
+        python=python,
+    )
+
+    target = unit_path(unit_name)
+    target.parent.mkdir(parents=True, exist_ok=True)
+    target.write_text(rendered, encoding="utf-8")
+    try:
+        target.chmod(0o600)
+    except OSError:
+        # chmod can fail on tmpfs/WSL-mounted Windows paths — non-fatal,
+        # the file is still in the user's own config dir.
+        pass
```

Ruff (0.15.12)

[warning] Use `contextlib.suppress(OSError)` instead of `try`-`except`-`pass`

(SIM105)